packages/react-ui-components/src/components/SchemaForm/Widgets/CheckboxesWidget/utils.ts

```diff
@@ -0,0 +1,92 @@
+import { IRadioButton } from '@kelvininc/ui-components';
+import { ALL_BUTTON_VALUE } from './config';
+import { ICheckboxConfig } from './types';
+
+export const buildRadioButtons = <T extends { label: string; value: string }>(options: T[], enumDisabled: string[], { multiple, allButton }: ICheckboxConfig): IRadioButton[] =>
+	options.reduce<IRadioButton[]>(
+		(accumulator, { label, value }) => [
+			...accumulator,
+			{
+				label,
+				value,
+				disabled: enumDisabled.indexOf(value) !== -1
```

Comment:
The disabled should include this condition `disabled={disabled || itemDisabled || readonly}` not only the itemDisabled

Reply:
Absolutely 👍